examples/internal/proto/examplepb/stream.pb.gw.go

```diff
@@ -131,20 +132,19 @@ func request_StreamService_BulkEcho_0(ctx context.Context, marshaler runtime.Mar
 	go func() {
 		for {
 			if err := handleSend(); err != nil {
+				errChan <- err
 				break
 			}
 		}
-		if err := stream.CloseSend(); err != nil {
-			grpclog.Errorf("Failed to terminate client stream: %v", err)
-		}
+		close(errChan)
```

Comment:
Could we do this in a defer at the top of the goroutine instead to make it even clearer?